tests/scripts/optdata/bootstrap.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+
+"""
+    This script prepares the local source tree to be built with
+    custom optdata. Simply run this script and follow the
+    instructions to inject manually created optdata into the build.
+"""
+
+import argparse
+import json
+import os
+from os import path
+import shutil
+import subprocess
+import sys
+
+# Display the docstring if the user passes -h|--help
+argparse.ArgumentParser(description=__doc__).parse_args()
+
+SCRIPT_ROOT = path.dirname(path.realpath(__file__))
+REPO_ROOT = path.realpath(path.join(SCRIPT_ROOT, '..', '..', '..'))
+
+ORIGIN_FILE = path.join(SCRIPT_ROOT, 'project.json')
+TARGET_FILE = path.join(REPO_ROOT, 'src', '.nuget', 'optdata', 'project.json')
+
+ARCH_LIST = ['x64', 'x86']
+TOOL_LIST = ['IBC', 'PGO']
+
+def get_buildos():
+    """Returns the Build_OS component used by the build system."""
+    if os.name == 'nt':
+        return 'Windows_NT'
+    else:
+        sysname = os.uname()[0]
+        return 'OSX' if sysname.casefold() == 'Darwin'.casefold() else sysname
```

Comment:
Can we just use sysname.lower() here? I know casefold is more aggressive than lower, but since all we are comparing to is a lowercase 'Darwin', I don't think we need that aggressive of a a lower, and casefold doesn't work in Python2 without importing extra libraries. If possible, I'd like this script to be as compatible with both Python2 and Python3 as possible since the main dependency in CoreCLR is on Python2.